### DISCUSSION TITLE
Web Client

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#5527] [Ideas]
The ability to have a setting to run podman desktop as a web server which can be accessed by the localhost or remotely if port forwarded.

For exmaple, lets say you have a laptop and a server. You install an container engine like podman and podman desktop on the server and enable the web server feature in podman desktop. Then on your laptop, you do not need to install anything and you can control podman with a GUI in your laptop's web browser by connecting to the podman desktop web server.

This can be a useful feature for when one installs docker engine on windows inside of WSL and then can also install podman desktop inside WSL and control it through the localhost.

https://github.com/containers/podman-desktop/issues/279#issuecomment-1817763216

**Comment by stevebovy** [comment]
Yes,  I agree 200%,   I have a vps  where using a gui  is cumbersome because it must function via  an rdp and/or 
some other remote desktop teck.

Two alternative suggestions:

Add deep integration support with portainer

Or   deep  integration support with dockge    

The cockpit support is  a "nice-try"    starting point,   but  simply not adequate in any good way.

I really like  the dockge  seamlessly integrated "text" based approach.   Because  their is no magically hidden data-base.   And the text is a well organized back up with seamless command line integration.

**Comment by dsferreira54** [comment]
This would be an excellent feature, it would be something similar to the web client that someone made for VSCode: https://medium.com/@taufiqpsumarna/run-vs-code-on-browser-b4320cc9d375
